client/src/views/users/GreetUserModal.vue

```diff
@@ -3,30 +3,49 @@
 <template>
   <ion-page class="modal-stepper">
     <ms-wizard-stepper
-      v-show="pageStep > GreetUserStep.WaitForGuest && pageStep < GreetUserStep.Summary"
+      v-show="pageStep > GreetUserStep.WaitForGuest && pageStep < GreetUserStep.Summary && isLargeDisplay"
       :current-index="getStepperIndex()"
       :titles="['UsersPage.greet.steps.hostCode', 'UsersPage.greet.steps.guestCode', 'UsersPage.greet.steps.contactDetails']"
     />
     <ion-button
       slot="icon-only"
       @click="cancelModal()"
       class="closeBtn"
-      v-show="pageStep !== GreetUserStep.Summary"
+      v-if="pageStep !== GreetUserStep.Summary && isLargeDisplay"
     >
       <ion-icon
         :icon="close"
         class="closeBtn__icon"
       />
     </ion-button>
     <div class="modal">
-      <ion-header class="modal-header">
+      <ion-header
+        class="modal-header"
+        v-if="isLargeDisplay"
+      >
         <ion-title class="modal-header__title title-h2">
-          {{ $msTranslate(getTitleAndSubtitle()[0]) }}
+          {{ $msTranslate(getStep(pageStep).title) }}
         </ion-title>
-        <ion-text class="modal-header__text body">
-          {{ $msTranslate(getTitleAndSubtitle()[1]) }}
+        <ion-text
+          v-if="getStep(pageStep).subtitle"
+          class="modal-header__text body"
+        >
+          {{ $msTranslate(getStep(pageStep).subtitle) }}
         </ion-text>
       </ion-header>
+      <small-display-step-modal-header
+        v-else
+        @close-clicked="cancelModal()"
+        :title="getStep(pageStep).title"
+        :subtitle="getStep(pageStep).subtitle"
+        :step="{
+          icon: personAdd,
+          title: 'HomePage.noExistingOrganization.joinOrganization',
```

Comment:
This locale does not match the user action: `Join an organization` on the greet/onboarding side.

There doesn't seem to be an existing locale that would fit, you can make one similar to `UsersPage.greet.titles.waitForGuest`.